tests/compat.sh

```diff
@@ -927,13 +969,16 @@ run_client() {
 # MAIN
 #
 
-if cd $( dirname $0 ); then :; else
-    echo "cd $( dirname $0 ) failed" >&2
-    exit 1
-fi
-
```

Comment:
Minor: Is it a MUST to move this to the beginning of the file?

Reply:
No, it's not required. Since we introduce `ORIGINAL_PWD`, I think it makes the code more readable if we move this part of code in the beginning of the file. Additionally, this will be identical with `ssl-opt.sh`.